Sources/Algorithm.swift

```diff
@@ -56,13 +56,13 @@ public extension StagedChangeset where Collection: RangeReplaceableCollection, C
             return
         }
 
-        // Returns the changesets that all deletions if source is not empty and target is empty
+        // Returns the changesets that all deletions if source is not empty and target is empty.
         if !sourceElements.isEmpty && targetElements.isEmpty {
             self.init([Changeset(data: target, elementDeleted: sourceElements.indices.map { ElementPath(element: $0, section: section) })])
             return
         }
 
-        // Returns the changesets that all insertions if source is empty and target is not empty
+        // Returns the changesets that all insertions if source is empty and target is not empty.
```

Comment:
Not fan of different comments - some of them have "Return`s`" and the next one "Calculate" (without `s`)